securebanking-openbanking-uk-rs-simulator-server/src/main/java/com/forgerock/securebanking/openbanking/uk/rs/api/discovery/DiscoveryApiService.java

```diff
@@ -54,13 +55,14 @@ public DiscoveryApiService(com.forgerock.securebanking.openbanking.uk.rs.api.dis
     /**
      * Builds a {@link Map} of Open Banking APIs that both the application and customer supports. The APIs are grouped
      * by "group" name (e.g. AISP, PISP) and listed by version.
+     *
+     * @return a {@link Map} of supported Open Banking APIs.
      */
-    @PostConstruct
```

Comment:
I can no longer build the list of URLs when starting up the service, as I'm relying on a request containing the X-Forwarded headers.